setup.py

```diff
@@ -45,7 +45,7 @@
     install_requirements = [
         "gbd_mapping>=3.1.0, <4.0.0",
         "vivarium>=2.1.1",
-        "vivarium_public_health>=2.0.0",
+        "vivarium_public_health>=2.2.0",
```

Comment:
For archival this should be pinned with ==, i.e. 
```suggestion
        "vivarium_public_health==2.2.0",
```